ccmbioinfo/lfs
# CARDS-2390: Rework the clarity import functionality to support multiple sources

To test:

- prems
    - build with `mvn clean install -Pdocker`
    - in `compose-cluster` run `python3 generate_compose_yaml.py --mssql --cards_project cards4prems --oak_filesystem --dev_docker_image --composum --adminer --server_address=localhost:8080`
    - in `compose-cluster/mssql` run `python generate_test_YE_sql.py -n 100 --time_spread_seconds 3456000 prems_sample.sql`
    - in `compose-cluster` start with `docker-compose build && docker-compose up`
    - at `http://localhost:1435/?mssql=mssql&username=sa&db=master&ns=path&import=` import the generated `prems_sample.sql`
    - Trigger the import at `http://localhost:8080/Subjects.importClarity`
    - Wait a bit, make sure that visits were imported
    - Stop docker and cleanup with `docker-compose rm -vf && ./cleanup.sh`
- proms
    - build with `mvn clean install -Pdocker`
    - in `compose-cluster` run `python3 generate_compose_yaml.py --mssql --cards_project cards4proms --oak_filesystem --dev_docker_image --composum --adminer --server_address=localhost:8080`
    - in `compose-cluster/mssql` run `python3 generate_test_proms_sql.py -n 20 proms_sample.sql`
    - in `compose-cluster` start with `docker-compose build && docker-compose up`
    - at `http://localhost:1435/?mssql=mssql&username=sa&db=master&ns=path&import=` import the generated `proms_sample.sql`
    - Trigger the import at `http://localhost:8080/Subjects.importClarity`
    - Wait a bit, make sure that visits were imported
    - Stop docker and cleanup with `docker-compose rm -vf && ./cleanup.sh`
